### PR TITLE
feat: add cream-lending stables yield adaptor

### PR DESCRIPTION
**Pull request by creamD3v**

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running cream-lending adapter: 
        
 
 Test Suites: 1 failed, 1 total
    Tests:       4 failed, 10 passed, 14 total
    Snapshots:   0 total
    Time:        0.289 s
    Ran all test suites.
    
    Nb of pools: 2
     
    
    Sample pools:
    ┌─────────┬──────────────────────────────────────────────┬────────────┬─────────────────┬────────┬────────┬─────────┐
    │ (index) │                     pool                     │   chain    │     project     │ symbol │ tvlUsd │ apyBase │
    ├─────────┼──────────────────────────────────────────────┼────────────┼─────────────────┼────────┼────────┼─────────┤
    │    0    │ '0x19b3bde6f02fc2c985947dcc1dc2a4cddfd43ee8' │ 'ethereum' │ 'cream-lending' │ 'USDT' │  NaN   │   NaN   │
    │    1    │ '0xc42d8f3d791c107c458b2fea025a36669b51fc5f' │ 'ethereum' │ 'cream-lending' │ 'USDC' │  NaN   │   NaN   │
    └─────────┴──────────────────────────────────────────────┴────────────┴─────────────────┴────────┴────────┴─────────┘
    }

**Comment by creamD3v** [comment]
test failed due to the invalid RPC endpoint

**Comment by slasher125** [comment]
pls run the tests locally, tvl, apy values are wrong. also, i recommend to add the additional lending/borrow fields (see readme)

**Comment by creamD3v** [comment]
image

ran successfully on local with `ETHEREUM_RPC` provided

**Comment by creamD3v** [comment]
test failed due to the failed multicall: https://github.com/DefiLlama/yield-server/actions/runs/7913341110/job/21600804721?pr=1186#step:4:70
which caused by the invalid `ETHEREUM_RPC` in the test
@slasher125

**Comment by slasher125** [comment]
k, tvlUsd = available liquidity. see readme for borrow related fields

**Comment by llamatester** [comment]
Error while running cream-lending adapter: 
        
 
 Test Suites: 1 failed, 1 total
    Tests:       4 failed, 10 passed, 14 total
    Snapshots:   0 total
    Time:        0.316 s
    Ran all test suites.
    
    Nb of pools: 2
     
    
    Sample pools:
    ┌─────────┬──────────────────────────────────────────────┬────────────┬─────────────────┬────────┬────────┬─────────┬────────────────┬────────────────┐
    │ (index) │                     pool                     │   chain    │     project     │ symbol │ tvlUsd │ apyBase │ totalSupplyUsd │ totalBorrowUsd │
    ├─────────┼──────────────────────────────────────────────┼────────────┼─────────────────┼────────┼────────┼─────────┼────────────────┼────────────────┤
    │    0    │ '0x19b3bde6f02fc2c985947dcc1dc2a4cddfd43ee8' │ 'ethereum' │ 'cream-lending' │ 'USDT' │  NaN   │   NaN   │       0        │      NaN       │
    │    1    │ '0xc42d8f3d791c107c458b2fea025a36669b51fc5f' │ 'ethereum' │ 'cream-lending' │ 'USDC' │  NaN   │   NaN   │       0        │      NaN       │
    └─────────┴──────────────────────────────────────────────┴────────────┴─────────────────┴────────┴────────┴─────────┴────────────────┴────────────────┘
    }

File: src/adaptors/cream-lending/index.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  ];
+  const prices = await getPrices(underlying_addresses);
+
+  return [
+    {
+      pool: CR_USDT.toLowerCase(),
+      chain: 'ethereum',
+      project: 'cream-lending',
+      symbol: 'USDT',
+      tvlUsd: (parseFloat(cashes[0]) / 10 ** 6) * prices[0],
```
Comment (creamD3v) [inline src/adaptors/cream-lending/index.js:78]: updated

**Comment by llamatester** [comment]
Error while running cream-lending adapter: 
        
 
 Test Suites: 1 failed, 1 total
    Tests:       4 failed, 10 passed, 14 total
    Snapshots:   0 total
    Time:        0.293 s
    Ran all test suites.
    
    Nb of pools: 2
     
    
    Sample pools:
    ┌─────────┬──────────────────────────────────────────────┬────────────┬─────────────────┬────────┬────────┬─────────┬────────────────┬────────────────┐
    │ (index) │                     pool                     │   chain    │     project     │ symbol │ tvlUsd │ apyBase │ totalSupplyUsd │ totalBorrowUsd │
    ├─────────┼──────────────────────────────────────────────┼────────────┼─────────────────┼────────┼────────┼─────────┼────────────────┼────────────────┤
    │    0    │ '0x19b3bde6f02fc2c985947dcc1dc2a4cddfd43ee8' │ 'ethereum' │ 'cream-lending' │ 'USDT' │  NaN   │   NaN   │       0        │      NaN       │
    │    1    │ '0xc42d8f3d791c107c458b2fea025a36669b51fc5f' │ 'ethereum' │ 'cream-lending' │ 'USDC' │  NaN   │   NaN   │       0        │      NaN       │
    └─────────┴──────────────────────────────────────────────┴────────────┴─────────────────┴────────┴────────┴─────────┴────────────────┴────────────────┘
    }

**Pull request merged**